ProjectSidewalk/SidewalkWebpage
# Resolves #1459: Keyboard shortcuts no longer log a click

Resolves #1459 

Triggering low level events with `{lowLevelLogging: false}` now skips logging their respective LowLevelEvent. I updated all instances of click triggering to pass this parameter.

I also noticed the modal mission "OK" shortcut was missing its own event, so I quickly added that.

To test:
- Ensure that keyboard shortcuts still function
- Ensure that keyboard shortcuts don't log LowLevelEvent_click
